### ISSUE TITLE
Post site launches clean up/url updating

### ISSUE THREAD

**Issue by annmarieven** [betheluniversity/cascade#699]
We need to go through and update links that are linking to the old site. Yes, they will be redirecting but we'd like to set the links up correctly as internal links now that the pages live in Cascade.

Start with this grid and check on anything in the "url updates/notes" column:
https://docs.google.com/spreadsheets/d/1638zkuIoM27Z0C2AN2YcBaEidc5GFHnDkkJoSUp_t7E/edit#gid=616838558

Can you also check the "still had link to old silva" column and see if the url is redirecting? If not, compile a list and submit it to Jmo.

Reassign this to me when you're done and I'll find more post-launch updates that need to happen. Thanks!

**Comment by annmarieven** [comment]
@emily-johnson is this done? 

Next item on the list - delete news articles in cascade 2011 and older 

Move all articles from this folder: https://editor.its.bethel.edu/silva/www/news/articles/edit/tab_edit to this folder: https://editor.its.bethel.edu/silva/archive/www/news/articles/edit/tab_edit

**Comment by emily-johnson** [comment]
I updated all of the url updates/notes from the google sheet and had Jmo do the redirects, and just now I deleted the 2008-2011 articles from Cascade.

@e-jameson I was trying to cut/paste in new silva and when I cut a page, the paste options appear on the page, but when I navigate away to the folder I want to paste it to, the paste option disappears. I was specifically trying to move the [fulbright-scholar](https://editor.its.bethel.edu/silva/www/news/articles/edit/tab_edit) into the 2012 September folder.

Help?
